.github/workflows/continuous-integration.yml

```diff
@@ -881,7 +907,7 @@ jobs:
         run: yarn cypress-mochawesome-to-bigquery
         working-directory: testing-tools-team-dashboard-data
         env:
-          IS_MASTER_BUILD: ${{ needs.cypress-tests-prep.outputs.is_master_build }}
```

Comment:
I don't have time to do a full review right now, but a bunch of the graphs in DOMO are generated from queries against `IS_MASTER_BUILD` being `true` or `false`:

![image](https://user-images.githubusercontent.com/9326247/156663239-5fb3739d-b378-4925-b72e-0aa78d027781.png)

These graphs will definitely be affected by `IS_MASTER_BUILD` always being `false`. Which is fine--they can be changed. Maybe we can talk about this tomorrow.

Thanks.

Reply:
Since we won't be running the full test suite in CI, `IS_MASTER_BUILD` needs to be `false` here. The full suite will run in the new `e2e-tests.yml` workflow, so I've set `IS_MASTER_BUILD` to `true` in that file: https://github.com/department-of-veterans-affairs/vets-website/pull/20413/files#diff-194218c48b9a0cdd03974145733804c2d992ca818529fe2fa69a501d8b5b1cc3R355